### DISCUSSION TITLE
Best way to group translation strings under issue numbers

### DISCUSSION THREAD

**Discussion by wotaen** [WeblateOrg/weblate#9752] [Q&A] [answered]
We'd like to organize translations under their issue number. We use version 4.x, currently I can add label, but that needs to be created first. I tried to add the issue number in explanation but when searching for `explanation:ISSUE-123` search came up with strings without this explanation too.

So is there any way we can simply add label/flag to translation string that's not predefined but rather created on the fly? Any other ideas how to organize new strings? Bearing in mind that there can be hundreds of issues.

**Comment by nijel** [comment]
The explanation search should work, please open an issue describing in more details how it behaves.

**Reply by wotaen** [reply]
I can see now, I was confusing the result with "Nearby strings" tab. Still though, can I get a list of results, instead of this:
![image](https://github.com/WeblateOrg/weblate/assets/715052/872b49ce-070b-405c-9215-fa082ba28437), where I have visually no idea what's in the result set?

**Reply by nijel** [reply, marked as answer]
Use /browse instead of the /translate URL.
